### PR TITLE
Fix latex equation in doc for 4FGL-DR3 SECPL model

### LINKED ISSUE #4312: Bad rendering of Super Exponential Cutoff Power Law Model used for 4FGL-DR3 equations in docs
**Gammapy version**
dev

**Bug description**

The latex rendering of equations in the following doc page is broken. 
https://docs.gammapy.org/dev/user-guide/model-gallery/spectral/plot_super_exp_cutoff_powerlaw_4fgl.html

I think that it is due to the `\a`that appears multiple times in `\frac`, but I am not really sure as I don't understand HTML.

### PR DESCRIPTION
**Pull request by QRemy**
Fix latex equation in doc for 4FGL-DR3 SECPL model.
  @MRegeard, this resolves #4312.

### CODE REVIEW

File: examples/models/spectral/plot_super_exp_cutoff_powerlaw_4fgl.py [hunk 1]
```diff
@@ -10,16 +10,7 @@
...
-    \phi(e) =
-            \begin{cases}
-                \phi_0 \cdot \left(\frac{E}{E_0}\right)^{\frac{\a}{\Gamma_2} -\Gamma_1} \cdot \exp \left(
-                  \frac{\a}{\Gamma_2^2} \left( 1 - \left(\frac{E}{E_0}\right)^{\frac{\a}{\Gamma_2} \right)
-              \right)&
-                              \\
-                \phi_0 \cdot \left(\frac{E}{E_0}\right)^{ -\Gamma_1 - \frac{\a}{2} \ln \frac{E}{E_0} - \frac{\a \Gamma_2}{6} \ln^2 \frac{E}{E_0} - \frac{\a \Gamma_2^2}{24} \ln^3 \frac{E}{E_0}}\\
-                0 & \text{for } \left| \Gamma_2 \ln \frac{E}{E_0}  \right|
-            \end{cases}
+\phi(e) = \begin{cases} \phi_0 \cdot \left(\frac{E}{E_0}\right)^{\frac{a}{\Gamma_2} -\Gamma_1} \cdot \exp \left( \frac{a}{\Gamma_2^2}\left( 1 - \left(\frac{E}{E_0}\right)^{\Gamma_2} \right) \right) \\ \phi_0 \cdot \left(\frac{E}{E_0}\right)^{ -\Gamma_1 - \frac{a}{2} \ln \frac{E}{E_0} - \frac{a \Gamma_2}{6} \ln^2 \frac{E}{E_0} - \frac{a \Gamma_2^2}{24} \ln^3 \frac{E}{E_0}} & \text{for } \left| \Gamma_2 \ln \frac{E}{E_0} \right| < 10^{-2} \end{cases}
```
Comment (bkhelifi) [inline examples/models/spectral/plot_super_exp_cutoff_powerlaw_4fgl.py:13]: It should be \phi(E), is not it?
Are the absolute value brackets properly printed?

**Review by AtreyeeS — COMMENTED** [review]
Thanks @QRemy ! If I am not mistaken, the function should be only Eq (3) of this paper, https://arxiv.org/pdf/2201.11184.pdf, no?
Currently, the formula is (eq 2)*(eq 3)

**Comment by QRemy** [comment]
> Thanks @QRemy ! If I am not mistaken, the function should be only Eq (3) of this paper, https://arxiv.org/pdf/2201.11184.pdf, no? Currently, the formula is (eq 2)*(eq 3)

No it's (3) for the condition and 2 otherwise as given here https://fermi.gsfc.nasa.gov/ssc/data/analysis/scitools/source_models.html#PLSuperExpCutoff4

Now it should look like this in gammapy 
![image](https://user-images.githubusercontent.com/25845654/217634252-24fe1fb5-4216-4d78-9c10-c726b2954a56.png

File: examples/models/spectral/plot_super_exp_cutoff_powerlaw_4fgl.py (hunk 1 above)
Comment (QRemy) [inline examples/models/spectral/plot_super_exp_cutoff_powerlaw_4fgl.py:13]:
```suggestion
\phi(E) = \begin{cases} \phi_0 \cdot \left(\frac{E}{E_0}\right)^{\frac{a}{\Gamma_2} -\Gamma_1} \cdot \exp \left( \frac{a}{\Gamma_2^2}\left( 1 - \left(\frac{E}{E_0}\right)^{\Gamma_2} \right) \right) \\ \phi_0 \cdot \left(\frac{E}{E_0}\right)^{ -\Gamma_1 - \frac{a}{2} \ln \frac{E}{E_0} - \frac{a \Gamma_2}{6} \ln^2 \frac{E}{E_0} - \frac{a \Gamma_2^2}{24} \ln^3 \frac{E}{E_0}} & \text{for } \left| \Gamma_2 \ln \frac{E}{E_0} \right| < 10^{-2} \end{cases}
```

Comment (QRemy) [reply examples/models/spectral/plot_super_exp_cutoff_powerlaw_4fgl.py]: -  yes
- yes, see the image below

**Review by AtreyeeS — APPROVED** [review]
Thanks @QRemy ! Perfect

**Review by registerrier — APPROVED** [review]
Thanks @QRemy. Looks good!

**Pull request merged**